ui/src/viewers/PdfViewer.jsx

```diff
@@ -291,6 +291,7 @@ const mapStateToProps = (state, ownProps) => {
   const pageQuery = baseQuery
     .set('highlight', true)
     .set('highlight_text', queryText)
+    .set('highlight_count', 15)
```

Comment:
@sunu Do you think this can have a significant impact on ES performance? Highlights need to be generated based on a single page only and 15 fragments is not a huge number, but I’m not sure how expensive highlighting is.

Reply:
@sunu What are your thoughts on this?